[velbus] New functionality and bug fix (#15661)
openhab/openhab-addons: bundles/org.openhab.binding.velbus/README.md

`[CHx="..."]` is optional, and represents the name of channel x, e.g. CH1 specifies the name of channel 1.

The `vmbel1`, `vmbel2`, `vmbel4`, `vmbelpir`, `vmbel1`, `vmbgp1`, `vmbgp1-2`, `vmbgp2`, `vmbgp2-2`, `vmbgp4`, `vmbgp4-2`, `vmbgp4pir` and `vmbgp4pir-2` thing type also supports 1 subaddress. This subaddress is the Sensor Address. The value "FF" = not configured.

```java
Thing velbus:vmbel1:<bridgeId>:<thingId> [subaddress1="A1"]
```

> The `vmbelo`, `vmbgpo`, `vmbgpod` and `vmbgpod-2` thing type also supports 4 subaddresses. The subaddress4 is the Sensor Address. The value "FF" = not configured.

Reviewer: Similarly:

```suggestion
The `vmbelo`, `vmbgpo`, `vmbgpod` and `vmbgpod-2` thing type also requires configuration of 4 subaddresses in order to access full functionality of the module. The `subaddress4` is the address of the thermostat channels. The value "FF" disables handling of the corresponding group of channels.
```

```java
Thing velbus:vmbelo:<bridgeId>:<thingId> [subaddress1="B1", subaddress2="B2", subaddress3="B3", subaddress4="B4"]
```

The `vmbdali` thing also supports 9 subaddresses. The value "FF" = not configured.

```java
Thing velbus:vmbelo:<bridgeId>:<thingId> [subaddress1="F1", subaddress2="F2", subaddress3="F3", subaddress4="F4", subaddress5="F5", subaddress6="F6", subaddress7="F7", subaddress8="F8", subaddress9="F9"]
```

> In the UI, the subaddress settings are in the advanced configuration of the module.

Reviewer: Is this still true? The `config.xml` does not contain `<advanced>true</advanced>` for the subaddress configuration.

Author: Whoops, I forgot to remove this during my last changes.

Author: Updated in last commit.

vmb1dm, vmb1led, vmb4dc, vmbdme, vmbdmi, vmbdmir
```

> The `vmbdali` thing has 16 virtual light. A virtual light is created from 4 channels of the DALI module.

Reviewer: ```suggestion
The `vmbdali` thing has 16 virtual light channels. A virtual light combines 3 or 4 VMBDALI module channels into an OpenHAB channel to control RGB or RGBW lights.
```

or something along those lines if I understood it correctly?

This is because an RGBW DALI light is configured on the VMBDALI module with 4 channels (Red channel, Green channel, Blue channel, White channel).  

> The configuration is set like this :

Reviewer: ```suggestion
The configuration is set like this:
```

```java
Thing velbus:vmbdali:<bridgeId>:<thingId> [VL1="R,G,B,W"]
```

> The white channel is optionnal.

Reviewer: ```suggestion
The white channel is optional.
```

e.g.:
```java

> Thing velbus:vmbdali:<bridgeId>:<thingId> [VL1="CH1,CH2,CH3,CH4", VL2="CH6,CH7,CH5"]

Reviewer: Since this is an example, fill in the rest of the placeholders as well?

```suggestion
Thing velbus:vmbdali:1:01 [VL1="CH1,CH2,CH3,CH4", VL2="CH6,CH7,CH5"]
```

Author: Updated in last commit.
